app/assets/javascripts/initializers/initializeRuntimeBanner.js

```diff
@@ -0,0 +1,58 @@
+/* global Runtime */
+
+function handleDismissRuntimeBanner(event) {
+  const runtimeBanner = document.querySelector('.runtime-banner');
+  if (runtimeBanner) {
+    runtimeBanner.remove();
+  }
+}
+
+// A separate function is needed for this because it's impossible to test a
+// custom scheme launch in Cypress. More about this here:
+// https://medium.com/cypress-io-thailand/understand-stub-spy-apis-that-will-make-your-test-better-than-ever-on-cypress-io-797cb9eb205a
```

Comment:
Small request: could we use JSDoc formatting for the functions in this file? It might make it easier for us to parse through and understand what's happening here, and hopefully that'll make it simpler to change this code in the future 👍🏽